components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -198,6 +202,11 @@ - (void)setTitleIcon:(UIImage *)titleIcon {
   [self.titleIconImageView sizeToFit];
 }
 
+- (void)setTitleIconTintColor:(UIColor *)titleIconTintColor {
+  _titleIconTintColor = titleIconTintColor;
+  self.titleIconImageView.tintColor = titleIconTintColor;
```

Comment:
Since self.titleIconImageView doesn't always exist, it might be worth wrapping this in an if to explicitly communicate this with engineers.
if (self.titleIconImageView) {
  self.titleIconImageView.tintColor = titleIconTintColor;
}

Reply:
Done.